Workbooks/Windows Virtual Desktop/NoData/NoData.workbook

```diff
@@ -345,7 +345,7 @@
               "links": [
```

Comment:
This need to be conditional visible using "UsingAMA" parameter
UsingAMA != no

Reply:
Added the condition back.